bioimageio_chatbot/chatbot.py

```diff
@@ -181,8 +183,24 @@ async def respond_to_user(question_with_history: QuestionWithHistory = None, rol
     )
     return customer_service
 
+def save_chat_history(session_id, chat_history, context):
+    # Create a folder with the session ID as the name
+    os.makedirs("chat_sessions", exist_ok=True)
+
+    # Create a chat_log.json file inside the session folder
+    chat_log_path = os.path.join("chat_sessions", f"{session_id}-chatlogs.json")
+
+    chat_his_dict = {'conversations':chat_history, 'timestamp': str(datetime.datetime.now().strftime("%Y-%m-%d %H:%M:%S")), 'user': context['user']}
+    # Serialize the chat history to a json string
+    chat_history_json = json.dumps(chat_his_dict)
+
+    # Write the serialized chat history to the json file
+    with open(chat_log_path, 'w', encoding='utf-8') as file:
```

Comment:
Would be better if we use aiofiles to write it: https://github.com/Tinche/aiofiles